libyara/grammar.y

```diff
@@ -1318,6 +1318,10 @@ regexp
 boolean_expression
     : expression
       {
+        // If the value of an expression is ever undefined we have a problem.
```

Comment:
I would say that an expression can be undefined, in fact, most expressions will be undefined because their value can't be known at compile time. I'm not understanding the reasoning behind this fix.

Reply:
It fixes a compiler crash. I don't think the fix is technically correct but I did want to bring it up as a fuzzer found it and it got auto assigned to me. ;)